indra-service/src/test/java/org/lambda3/indra/service/test/VectorResourceImplTest.java

```diff
@@ -0,0 +1,65 @@
+package org.lambda3.indra.service.test;
+
+
+import org.apache.commons.math3.linear.RealVector;
+import org.lambda3.indra.client.ScoreFunction;
+import org.lambda3.indra.core.IndraDriver;
+import org.lambda3.indra.core.Params;
+import org.lambda3.indra.core.VectorSpaceFactory;
+import org.lambda3.indra.core.composition.VectorComposition;
+import org.lambda3.indra.core.test.IndraDriverTest;
+import org.lambda3.indra.core.translation.IndraTranslatorFactory;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+
+/*-
+ * ==========================License-Start=============================
```

Comment:
The copyright header should be between package declaration and imports. Use `mvn -Plicenses process-resources` to auto generate instead of copy&paste, please.